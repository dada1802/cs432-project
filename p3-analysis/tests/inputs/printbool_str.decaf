def int main()
{
    print_bool("buyue box");
    return 0;
}