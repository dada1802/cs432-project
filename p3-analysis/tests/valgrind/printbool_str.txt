==700880== Memcheck, a memory error detector
==700880== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==700880== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==700880== Command: ../decaf inputs/printbool_str.decaf
==700880== 
Type mismatch in parameter 0 of call to 'print_bool': expected bool but found str on line 3
==700880== 
==700880== HEAP SUMMARY:
==700880==     in use at exit: 0 bytes in 0 blocks
==700880==   total heap usage: 1,801 allocs, 1,801 frees, 215,368 bytes allocated
==700880== 
==700880== All heap blocks were freed -- no leaks are possible
==700880== 
==700880== For lists of detected and suppressed errors, rerun with: -s
==700880== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
