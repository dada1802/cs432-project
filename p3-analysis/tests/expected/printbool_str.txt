Type mismatch in parameter 0 of call to 'print_bool': expected bool but found str on line 3
